minecraft/franchise/internal/attr.go

```diff
@@ -0,0 +1,7 @@
+package internal
+
+import "log/slog"
+
+const errorKey = "error"
+
+func ErrAttr(err error) slog.Attr { return slog.Any(errorKey, err) }
```

Comment:
This doesn't seem like a particularly useful function. Definitely wouldn't put it in its own file.